### DISCUSSION TITLE
오늘 접종 하고 왔습니다.

### DISCUSSION THREAD

**Discussion by koolingo** [changdoc/naver-vaccine-macro#97] [General]
개발자님 덕분에 오늘 드됴 성공해서 접종 하고 왔습니다.
정말 감사 드립니다!!! 
다른분들도 꼭 성공하시길 빌며 제 경험담 남깁니다. 

1. 지난주 토요일 유튜브에 있는 네이버 지도 기반으로 하는 매크로 사용
2.  naver-vaccine-macro 이번주 부터 사용 (다중으로 이용시 차단된 후 번갈아서 사용함)
3. 어제부터 카카오 기반으로 하는 korea-covid-19-remaining-vaccine-macro 매크로와 병행
4. 오늘 오전에 naver-vaccine-macro 성공! (카카오도 동시 예약시도 했으나 실패!)
5.  
- 일주일동안 계속 실패 했으나 특정병원이 특정 시간대 알람이 온다는걸 메모 한후에
   특정시간대에는 naver-vaccine-macro를 통해 그 병원만 집중공략함!

   -  주변 병원이 언제 잔여백신 알람을 등록하는지 파악후 그 특정 시간대를 공략하는게 현재로썬 베스트!
    비슷한 시간대 알람이 오는 주변 병원이 많을경우는 가장 확률적으로 높은 병원을 선택하고
     (제 경험상 한 병원을 선택 하는건 정말 운 인듯.... 정작 등록한 병원은 알람 안오고 다른병원에서 휴대폰 알람이 계속 온적도....)
    다른 주변 병원들은 카카오 기반으로 하는 매크로를 사용해서 동시에 진행하는게 확률적으로 높음.

    - 네이버&카톡을 통해 예약 알람 신청을 시간대별로 바꿔가며 등록 
      (같은병원도 요일별 접종시간 다르니 그것도 미리체크!), (실시간으로 대기중, 마감, 없음으로 변동되니 그것도 계속 파악!)
       (휴대폰을 통한 알람신청후 잔여백신 예약하기는 카카오톡이 훨씬 편함)

**Comment by seokwonchoisnu** [comment]
저는 3일 6번 트라이만에 1시까지인 병원 12시 50분에 성공해서 차타고 바로 오늘 드디어 맞았습니다.

작은 병원이나 점심시간 계산하고 접종시간이 일찍끝나는 병원부터 한다음 차례 차례 접종 시간이 늦은 병원으로 바꾸면서 하시는게 좋을 거 같습니다.

개발자님 감사합니다.
